client/utils/getImageUrl.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+const getImageUrl = (productId) => new Promise((resolve, reject) => {
+  axios.get(`/api/products/${productId}/styles`)
+    .then((reviews) => {
+      const { results } = reviews.data;
+      const { photos } = results[0];
+      const firstPhotoUrl = photos[0].url;
```

Comment:
Nice use of promises. My question is we will need the ability to choose other than just the first photo in the future?

Reply:
Definitely a good question. It would be nice if you and I only had to make one API request to this endpoint. The way it stands, it seems we have to do it twice.